pyccel/stdlib/STC_Extensions/List_extensions.h

```diff
@@ -43,7 +43,26 @@ static inline i_key _c_MEMB(_max)(const Self* self) {
     }
     return max_val;
 }
+
 #endif
+
+// Function to insert value before position idx
+static inline void _c_MEMB(_insert)(Self* self, intptr_t idx, i_key value) {
```

Comment:
Why write a new method instead of using STC's `vec_X_insert_at` ?

Reply:
because the STC function does not allow for negative indices. Do you wanna add a codeblock to calculate the appropriate index and just call their function?